workspace_README.md

```diff
@@ -0,0 +1,124 @@
+   `this.samples.read2` respectively. Be sure to make the table name plural by adding an `s` after it (`sample` vs. 
+   `samples`). 
+5. The rest of the inputs can be configured as desired.
+6. Once all inputs are configured, you can click "Save" and then "Launch" to start the workflow. If everything was 
+   configured correctly, you'll see "You are launching 1 workflow run in this submission." in the popup. If you see 
+   that more than one workflow is being launched, go back through the configuration steps and ensure that a "set" of 
+   samples has been selected, as this workflow is designed to run once per dataset. 
+7. After launching, you can monitor the progress of the workflow in the "Submission History" tab. By default, Terra 
+   only displays workflows that have been launched in the past 30 days. If you want to see submission history from 
+   all time, make sure you select "All submissions" from the Date range drop down at the top of the page. 
+
+## Cost
+
+Although we can’t provide exact cost estimates for your specific dataset, the table below summarises example costs from running the full pipeline on a set of real datasets, varying both the number of targets and the number of samples. The cost shown reflects the total compute cost for each complete pipeline run across the full batch of samples, and then cost per sample (Total Cost/ Samples).
+
+| Panel Type               | Samples | Time                | Total Cost | Cost / Sample |
+|--------------------------|---------|---------------------|------------|---------------|
+| PfPHAST (57 targets)     | 217     | 2 hours 43 minutes  | $1.17      | $0.005        |
+| PfPHAST (57 targets)     | 1,152   | 9 hours 41 minutes  | $7.42      | $0.006        |
+| MAD4HatTeR (243 targets) | 96      | 3 hours 28 minutes  | $1.01      | $0.01         |
+| MAD4HatTeR (243 targets) | 149     | 4 hours 53 minutes  | $1.55      | $0.01         |
+| MAD4HatTeR (243 targets) | 506     | 13 hours 2 minutes  | $7.02      | $0.01         |
+| MAD4HatTeR (243 targets) | 862     | 25 hours 13 minutes | $24.02     | $0.03         |
+
+## Pre-configured Panels
+
+The pipeline supports several pre-configured panels. Specify your panel using the `pools` parameter when running the workflow.
+
+**MAD4HatTeR** (modular - combine pools as needed):
+- Available pools: `D1.1` (or `D1`), `R1.1` (or `R1`), `R2.1`, `R1.2` (or `R2`), `v1`, `v2`
+- Most common: `["D1","R1","R2"]`
+
+**PfPHAST** (modular):
+- Available pools: `M1.1` (or `M1`), `M2.1` (or `M2`), `M1.addon`
+
+**Other panels:**
+- `4cast`
+- `ama1`
+
+**Custom panels:** If your panel isn't listed, you can provide your own panel configuration files. Use the `amplicon_info_files` parameter along with either `genome` (whole genome reference) or `refseq_fasta` (targeted reference). See [example panel configurations](https://github.com/EPPIcenter/mad4hatter/tree/develop/panel_information) for reference.
```

Comment:
It is currently set up so that if it is a custom pool they can give. 
A. `amplicon_info_files` and `genome`
B. `amplicon_info_files` and `refseq_fasta`
C. `amplicon_info_files` and `targeted_reference_files`